torch/fx/experimental/proxy_tensor.py

```diff
@@ -1766,13 +1784,15 @@ def trace_subgraph(self, f: Callable, *args: object) -> GraphModule:
         # Create a new tracer based on parent's config
         sub_tracer = _MakefxTracer(
             self.decomposition_table,
-            self.tracing_mode,
+            "real",
             self._allow_non_fake_inputs,
             self.pre_dispatch,
             self.record_module_stack,
             self._allow_fake_constant,
             self._error_on_data_dependent_ops
         )
+        if isinstance(self.proxy_mode, ProxyTorchDispatchMode):
+            assert not self.proxy_mode.sym_mode.enable_tracing
```

Comment:
BTW, it doesn't actually work to disable sym_mode tracing here, because there's other logic inside the HOP's proxy tensor implementation that can also trigger thunk forces